doc/radosgw/metrics.rst

```diff
@@ -117,11 +98,51 @@ For a user ``User1`` doing operations on ``Bucket1``, the RGW Op Metrics can be
         ...
     ]
 
+Enabling the Radosgw User & Bucket Counter Caches
+=================================================
+
+To track op metrics by user the Radosgw the config value ``rgw_user_counters_cache`` must be set to ``true``. 
+
+To track op metrics by bucket the Radosgw the config value ``rgw_bucket_counters_cache`` must be set to ``true``. 
+
+These config values are set in the Ceph configuration file under the ``[client.rgw.{instance-name}]`` section and must be set before the radosgw is started or restarted to take effect.
+
+The counters in these caches live in memory. If the radosgw is restarted or crashes, all the  counters in the cache are lost.
+
+Radosgw User & Bucket Counter Cache Size & Eviction
+===================================================
+
+The user and bucket counter cache sizes should be sized appropriately depending on the expected number of users and buckets in the cluster.
+
+To help calculate the memory usage of a cache, it should be noted that each cache entry, encompassing all of the op metrics, is 1360 bytes. Both ``rgw_user_counters_cache_size`` and ``rgw_bucket_counters_cache_size`` can be used to set number of entries in each cache.
```

Comment:
the `1360` value will change as we add or remove counters here. please make it clear that this is a ballpark figure